composer.json

```diff
@@ -33,11 +33,13 @@
         "contao/imagine-svg": "^0.2.2 || ^1.0",
         "dantleech/phpcr-migrations-bundle": "^1.0",
         "doctrine/annotations": "^1.2",
-        "doctrine/common": "^2.7.1",
-        "doctrine/data-fixtures": "^1.1",
-        "doctrine/dbal": "^2.6",
+        "doctrine/collections": "^1.0",
+        "doctrine/cache": "^1.0",
+        "doctrine/data-fixtures": "^1.2",
```

Comment:
Related to: https://github.com/doctrine/data-fixtures/issues/213 fixed in (1.2): https://github.com/doctrine/data-fixtures/pull/222/files

Reply:
Needed 1.2.1 because of a circular reference error which was fixed in 1.2.1